### PR TITLE
Add merge triggers to official yaml

### PR DESCRIPTION
**Pull request by JoeRobich**
Move merge triggers into yaml.

### CODE REVIEW

**Review by RikkiGibson — APPROVED** [review]

File: azure-pipelines-official.yml [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# Branches that trigger a build on commit
+trigger:
```
Comment (genlu) [inline azure-pipelines-official.yml:2]: So i guess this work on both pipelines? Do we want to run it on devdiv only fro now (is it even possible to use condition here)?

Comment (JoeRobich) [reply azure-pipelines-official.yml:2]: If you've configured the new pipeline the same as the old, then you are already overriding this section.

**Comment by JoeRobich** [comment]
not sure this will work for us. Since all old branches are missing this section then they will trigger signed builds when they are updated. This is true for merge PR branches as I just had to cancel and delete like 20 signed builds because the merge bot created the branch and the branch didn't include a `triggers:` section.

**Comment by JoeRobich** [comment]
This PR is no longer necessary.

**Pull request closed without merging**